bgpd/bgp_vty.c

```diff
@@ -260,9 +260,12 @@ bgp_vty_find_and_parse_afi_safi_vrf (struct vty *vty, struct cmd_token **argv, i
   if (*vrf == VRF_UNKNOWN)
     {
       vty_out (vty, "View/Vrf specified is unknown: %s", vrf_name);
+      *idx = 0;
```

Comment:
because it's the error case? We were unable to figure out what vrf was specified( the last line in the doc )